src/EfuncMsg.chpl

```diff
@@ -42,6 +42,100 @@ module EfuncMsg
       :throws: `UndefinedSymbolError(name)`
       */
 
+
+//  This section is a rewrite of trig and hyp functions in new interface.
+//  This comment will be updated as other functions are rewritten, and deleted
+//  once the rewrite is complete.
+
+//  Error-throwing versions of each function are indented below their "parents"
+//  in order to improve readability.
```

Comment:
I think the chpl check linter is gonna pitch a fit over this indentation choice. For the sake of me not having tons of yellow squiggly underlines, do u mind having them indented at the same level?

Reply:
It's passed the upstream check multiple times, but sure, I can do that.

Reply:
Okay, git and I are not being friends again.  I made that change, and pushed it.  No, git said, everything is up-to-date.  Okay, I force pushed it.  Again, git says everything is up-to-date.  But the changes aren't here on the PR, and aren't on the branch on my repository either.  What am I doing wrong?

Reply:
I did that a few days ago, but I see the comment remains.  I may have to change that upstream, since I'm having issues on my local branches.